mscore/preferences.cpp

```diff
@@ -1154,13 +1154,13 @@ void PreferenceDialog::resetShortcutClicked()
     
 void PreferenceDialog::saveShortcutListClicked()
       {
-      QString saveFileName = QFileDialog::getSaveFileName(new QWidget(), "Save Shortcuts", "", tr("XML Files(*.xml)"));
+      QString saveFileName = QFileDialog::getSaveFileName(this, tr("Save Shortcuts"), "", tr("MuseScore Shortcuts File") + " (*.xml)");
```

Comment:
Still to be done: please compare the other places where we load and save files, esp. look at what is done if `preferences.nativeDialogs` is false (like on Linux I believe), as it had been mentioned on IRC yesterday

You could (and provbaly should) factor ot the getFilename stuff, e.g. like it is done for Style files